src/__tests__/starWarsSchema.js

```diff
@@ -19,6 +20,8 @@ import {
 
 import { getFriends, getHero, getHuman, getDroid } from './starWarsData.js';
 
+import type {Character, Human, Droid} from './starWarsData.js';
```

Comment:
stylistic issue: please use spaces inside the `{ }` in import to match the others in the project